### PR TITLE
Add German translation and one-sentence description of rights statements

### PR DESCRIPTION
**Pull request by anarchivist**
Ref rightsstatements/rightsstatements.github.io#131
Closes rightsstatements/data-model#55

### CODE REVIEW

**Review by aisaac — DISMISSED** [review]
Great job!
The only issues I could find are some extra whitespaces:
l126: Schutzrechte gestatten. """@de ;
l186: Rechteinhaber/s. """@de ;
l216: Rechteinhaber/s. """@de

**Comment by aisaac** [comment]
@anarchivist this is good for me now! Thanks a lot.
Should I merge it myself or do you want that @musebrarian have a look first?

**Comment by anarchivist** [comment]
Fine by me either way!

File: rights-statements.ttl [hunk 1]
```diff
@@ -129,16 +170,28 @@ You are free to use this Item in any way that is permitted by the copyright and
 
 <InC-EDU/1.0/> a dcterms:RightsStatement, skos:Concept ;
   skos:prefLabel "In Copyright - Educational Use Permitted"@en ;
+  skos:prefLabel "Urheberrechtsschutz – Nutzung zu Bildungszwecken erlaubt"@de ;
+  dcterms:description "This Rights Statement indicates that the Item labeled with this Rights Statement is in copyright but that educational use is allowed without the need to obtain additional permission."@en ;
+  dcterms:description "Dieser Rechtehinweis besagt, dass Urheberrechtsschutz besteht, aber die Nutzung des Objekts zu Bildungszwecken ohne zusätzliche Rechteeinholung gestattet ist."@de ;
   dcterms:modified "2015-10-16"^^xsd:date ;
```
Comment (musebrarian) [inline rights-statements.ttl]: @anarchivist  does this need an updated dcterms:modified date?

**Review by musebrarian — DISMISSED** [review]
Thanks @anarchivist!
Just one quick check whether "In Copyright - Educational Use Permitted" may need an updated modified date?

**Review by musebrarian — APPROVED** [review]

**Comment by aisaac** [comment]
OK I'm merging!

**Pull request merged**